docs/source/in5min.rst

```diff
@@ -0,0 +1,124 @@
+.. _in5min:
+
+PIConGPU in 5 Minutes
+=====================
+
+A guide to run, but not understand PIConGPU.
+It is aimed at existing and preconfigured high performance computing clusters (“supercomputers”, also: HPC systems).
+For more information on the individual steps please refer to the manual.
+
+This guide needs **shell access** (probably via ``ssh``) and **some auxilliary programs** (e.g. ``git``).
+Consider getting familiar with the shell and git.
+Please also read the tutorial for your local HPC cluster.
+
+We will use the following directories:
+
+- ``~/src/picongpu``: source files from github
+- ``~/picongpu.profile``: load the dependencies for your local environment
+- ``~/picongpu-projects``: scenarios to simulate
+- ``/scratch/mydir``: result data of the simulation runs
+
+Please replace them whenever appropriate.
+
+Get the source
+--------------
+
+Use git to obtain the source and use the current ``dev`` branch and put it into ``~/src/picongpu``::
+
+  mkdir -p ~/src
+  git clone -b dev https://github.com/ComputationalRadiationPhysics/picongpu ~/src/picongpu
+
+Setup
+-----
+
+You need :ref:`a lot of dependencies <install-dependencies>`.
+
+If you are on a known cluster you can take a shortcut:
+Select a profile file (``NAME.profile.example``) for your local HPC cluster from ``etc/picongpu`` and copy it to your home directory::
+
+  cp ~/src/picongpu/etc/picongpu/hemera-hzdr/k80_picongpu.profile.example ~/picongpu.profile
+
+This profile determines which part of the HPC cluster (“partition”, also: “queue”) – and thereby the type of GPUs – you will use.
+
+**Maybe you have to adjust this profile file.**
+Use your favorite editor.
+If unsure use nano: ``nano ~/picongpu.profile`` (Save with Ctrl+O, exit with Ctrl+X)
+Read the file (it’s short) and apply changes if appropriate.
+
+For this guide, add after the last line::
+
+  export SCRATCH=/scratch/mydir
+
+This is the location where runtime data and all results will be stored.
+Obviously change the path according to your local setup.
+Consult the documentation of your HPC cluster where to save your data.
+**On HPC clusters this is probably not your home directory.**
+
+Now activate your profile::
+
+  source ~/picongpu.profile
+
+You will have to repeat the last command every time you want to use PIConGPU.
+
+Create a Scenario
+-----------------
+
+Create a directory and copy an example::
+
+  mkdir -p ~/picongpu-projects/tinkering
+  pic-create $PIC_EXAMPLES/LaserWakefield ~/picongpu-projects/tinkering/try01
+  cd ~/picongpu-projects/tinkering/try01
+
+Usually you would now adjust the files in the newly created directory ``~/picongpu-projects/tinkering/try01`` – for this introduction we will use the parameters as provided.
+Also note how the variable ``$PIC_EXAMPLES`` has been provided becauses you executed ``source ~/picongpu.profile`` in the previous step.
```

Comment:
Perhaps one could also state something similar for `pic-create`: that is only available because the systems `$PATH` included t after sourcing. (This is the most common error when people want to create their second example and wonder why `pic-create` does not exist.)

Reply:
addressed in d0cebfcb5

Note: I don't mention `$PATH` because I think how searching for binaries works is too much information if you're unfamiliar with Linux in general.